test/acceptance/stub_module_method_test.rb

```diff
@@ -88,25 +24,6 @@ def self.my_module_method
     assert_equal 0, mod.mocha.__expectations__.length
   end
 
-  def test_should_be_able_to_stub_a_superclass_method
```

Comment:
Deleted since it's checking nothing more than stubbing of a non-existent method. The module (mod) doesn't 'inherit' the supermod singleton method simply by including supermod.